### ISSUE TITLE
Add tutorials for hk

### ISSUE THREAD

**Issue by core-man** [seismo-learn/software#137]
https://blog.seisman.info/hk-install/

**Comment by seisman** [comment]
Although hk is very popular, it requires very old GMT versions (GMT 4). I'm not sure if we should document it.

**Comment by core-man** [comment]
> Although hk is very popular, it requires very old GMT versions (GMT 4). I'm not sure if we should document it.

Yes, but we may only use it to compute RF which does not need gmt. We can revise the makefile to.ignore hk

**Comment by seisman** [comment]
There may exist better tools for calculating receiver functions (https://rf.readthedocs.io/en/latest/).

**Comment by core-man** [comment]
> There may exist better tools for calculating receiver functions (https://rf.readthedocs.io/en/latest/).

Yes, I kown. But we can run rfcnv in cml to compute rf. it is one line, similar to distaz.

**Comment by seisboy** [comment]
Hi, bro. Maybe I can try to write some tutorials for _hk1.3_  package.

**Comment by core-man** [comment]
> Hi, bro. Maybe I can try to write some tutorials for _hk1.3_ package.

@seisboy Thanks a lot. You can refer to https://seismo-learn.org/contributing/ to learn how to contribute to seismo-learn, e.g., reStructuredText basics, PR workflow. You can use the template (https://seismo-learn.org/software/template/）or any other software (e.g., https://seismo-learn.org/software/taup/). Just copy those files and revise them.

You may finish this issue with several PRs, e.g.,
- metadata page like https://seismo-learn.org/software/fk/
- introduction page like https://seismo-learn.org/software/fk/introduction/
- how to install page like https://seismo-learn.org/software/fk/install/
- more guides about how to use it. You may decide how many tutorials about how to use hk. It's better to discuss with us before you work on this part.
